resources/db/schema/mysql-ddl.sql

```diff
@@ -1087,7 +1087,7 @@ CREATE TABLE user_movies (
   id          INT(16) UNSIGNED               NOT NULL AUTO_INCREMENT,
   user_id INT(16)                        NOT NULL,
   imdbid      MEDIUMINT(7) UNSIGNED ZEROFILL NULL,
-  categoryid  VARCHAR(64)                    NULL DEFAULT NULL,
+  categories_id  VARCHAR(64)                    NULL DEFAULT NULL,
```

Comment:
I think it would be best represented as categories_ids?  Not sure if we have a standard for a column that would house multiple FKs at once to a table.